exec/fixedosc_llhscan.cc

```diff
@@ -207,19 +207,28 @@ void fixedosc_llhscan(const std::string &fitConfigFile_,
 
     // If it's the reactor PDF, two things are different from the others. Firstly we use the BuildOscillatedDist rather than just Build.
     // Also, we're going to loop over oscillation scan points to make the reactor's PDF for that point as well
+    std::vector<double> constrMeans_Map;
     if (it->first == "reactor_nubar")
     {
       reacPDFNum = pdfs.size();
       // Build the distribution with oscillation parameters at their nominal values
       // Pass double by reference here to get ratio of unoscillated to oscillated events
       double ratio = 1.0;
       dist = DistBuilder::BuildOscillatedDist(it->first, num_dimensions, pdfConfig, dataSet, deltam21_nom, theta12_nom, indexDistance, ratio);
-
       // Now we will scale the constraint on the unoscillated reactor flux by the ratio of the oscillated to unoscillated number of events
-      constrMeans[it->first] = constrMeans[it->first] * ratio;
-      constrSigmas[it->first] = constrSigmas[it->first] * ratio;
-      noms[it->first] = noms[it->first] * ratio;
-
+      
+      double constrMeans_config = constrMeans[it->first];  
+      double constrSigmas_config = constrSigmas[it->first];  
+      double noms_config        = noms[it->first];  
+      double min_config        = mins[it->first];  
+      double max_config        = maxs[it->first];  
+      constrMeans[it->first] = constrMeans_config * ratio;
+      constrSigmas[it->first] = constrSigmas_config * ratio;
+      noms[it->first] = noms_config * ratio;
+      mins[it->first] = min_config * ratio;
+      maxs[it->first] = max_config * ratio;
```

Comment:
is there a reason to define new variables here? Can't we just do `noms[it->first] = noms[it->first] * ratio;` and the same for mins, maxs, constrMeans and constrSigmas?

Just thinking, there's also a chance that once we apply oscillations with non-nominal values that the nom could become outside the min-max range. That's not a problem for this PR though!